app/assets/javascripts/hyrax/app.js.erb

```diff
@@ -135,7 +135,7 @@ Hyrax = {
     // Per Page select will submit its form to change records shown
     perPage: function () {
         var PerPage = require('hyrax/per_page');
-        $('#per_page').each(function () {
+        $('#sort, #per_page').each(function () {
```

Comment:
This seems incorrect to initialize the PerPage with a `#sort` identifier.  Can we change the name of `PerPage` to `SortAndPerPage` or `SearchOptions`?